docs/advanced-features/static-html-export.md

```diff
@@ -11,65 +11,62 @@ description: Export your Next.js app to static HTML, and run it standalone witho
+## Usage
 
-> `next export` is intended for scenarios where **none** of your pages have server-side or incremental data requirements (though statically-rendered pages can still [fetch data on the client side](/docs/basic-features/data-fetching.md#fetching-data-on-the-client-side)).
->
-> If you're looking to make a hybrid site where only _some_ pages are prerendered to static HTML, Next.js already does that automatically for you! Read up on [Automatic Static Optimization](/docs/advanced-features/automatic-static-optimization.md) for details.
->
-> `next export` also causes features like [Incremental Static Generation](/docs/basic-features/data-fetching.md#fallback-true) and [Regeneration](/docs/basic-features/data-fetching.md#incremental-static-regeneration) to be disabled, as they require [`next start`](/docs/api-reference/cli.md#production) or a serverless deployment to function.
-
-## How to use it
-
-Develop your app as you normally do with Next.js. Then run:
-
-```bash
-next build && next export
-```
-
-For that you may want to update the scripts in your `package.json` like this:
+Update your build script in `package.json` to use `next export`:
 
 ```json
 "scripts": {
   "build": "next build && next export"
 }
 ```
 
-And run it with:
+Running `npm run build` will generate an `out` directory.
 
-```bash
-npm run build
-```
+With `next export`, we build an HTML version of your app. At export time, we call [`getStaticProps`](/docs/basic-features/data-fetching.md#getstaticprops-static-generation) for each page that exports it, and pass the result to the page's component. `next export` will generate an HTML file for each page in your `pages` directory (or more for [dynamic routes](/docs/routing/dynamic-routes.md), where it will call [`getStaticPaths`](/docs/basic-features/data-fetching.md#getstaticpaths-static-generation) and generate pages based on the result).
 
-Then you'll have a static version of your app in the `out` directory.
-
-By default `next export` doesn't require any configuration.
-It will output a static HTML file for each page in your `pages` directory (or more for [dynamic routes](/docs/routing/dynamic-routes.md), where it will call [`getStaticPaths`](/docs/basic-features/data-fetching.md#getstaticpaths-static-generation) and generate pages based on the result).
 For more advanced scenarios, you can define a parameter called [`exportPathMap`](/docs/api-reference/next.config.js/exportPathMap.md) in your [`next.config.js`](/docs/api-reference/next.config.js/introduction.md) file to configure exactly which pages will be generated.
 
-## Deployment
+## Supported Features
+
+The majority of core Next.js features needed to build a static site are supported, including:
+
+- [Dynamic Routes](/docs/routing/dynamic-routes.md)
```

Comment:
When the page is generated using automatic static generation it requires additional configuration to route `/blog/:slug` to `out/blog/[slug].html`.

I think it should probably be `Dynamic Routes when using getStaticPaths`. which works in all cases.